lib/google_scraping/dashboard/queries/keyword_query.ex

```diff
@@ -0,0 +1,10 @@
+defmodule GoogleScraping.Dashboard.Queries.KeywordQueries do
```

Comment:
Module name should match the filename
```suggestion
defmodule GoogleScraping.Dashboard.Queries.KeywordQuery do
```